pull_request_template.md

```diff
@@ -0,0 +1,20 @@
+<!-- Add a meaningful description of the changes you made -->
```

Comment:
Is that comment visible to the user who creates PR? Or only to those who edit the template?

What was your intention?